src/main/client/src/dashBoard/bookList/book/Book.js

```diff
@@ -94,6 +95,16 @@ export default class Book extends Component {
             Edit
           </Button>
         )}
+        <Popup
+          className={styles.availablePopup}
```

Comment:
Can these classes be ids instead? Don't really like the idea of having empty classes